docs/migrate.rst

```diff
@@ -446,7 +446,7 @@ If you need lap functionality, the following shell script can be used instead::
 
     for lap in $(seq 1 ${RALLY_LAPS})
     do
-      esrally --pipeline=benchmark-only --user-tag lap:$lap
+      esrally --pipeline=benchmark-only --user-tags lap:$lap
```

Comment:
It would be very nice if we could document at the top of this file that `--user-tag` is now deprecated in favor of `--user-tags`. The next version can be found in https://github.com/elastic/rally/blob/master/esrally/_version.py#L1 so it'll be effective from `2.6.1`.

Reply:
As mentioned in https://github.com/elastic/rally/pull/1599#issuecomment-1288801857 maybe it's just enough to mention that starting with 2.6.1 `--user-tags` has been introduced as a "synonym" to `--user-tag`.

Reply:
oh, sorry I didn't notice this was a migration guide.
I think that might be better to be in the migration guide section for v2.6.1, so I simply reverts this change.